# Filter the interpolated setpoints from the trajectory with the list of actual setpoints.
string gait
string subgait
string version
string description
duration duration
ActualSetpoint[] actual_setpoints
trajectory_msgs/JointTrajectory joint_trajectory
Sound[] sounds